docs/api/classifier/threshold.rst

```diff
@@ -5,7 +5,7 @@ Discrimination Threshold
 
 .. caution:: This visualizer only works for *binary* classification.
 
-A visualization of precision, recall, f1 score, and "queue rate" with respect to the "discrimination threshold" of a binary classifier. The *discrimination threshold* is the probability or score at which the positive class is chosen over the negative class. Generally this is set to 50% but the threshold can be adjusted to increase or decrease the sensitivity to false positives or to other application factors.
+A visualization of precision, recall, f1 score, and queue rate with respect to the discrimination threshold of a binary classifier. The discrimination threshold is the probability or score at which the positive class is chosen over the negative class. Generally, this is set to 50% but the threshold can be adjusted to increase or decrease the sensitivity to false positives or to other application factors.
```

Comment:
I removed superfluous quotation marks in the text.

Reply:
Neither queue rate nor discrimination threshold is a term of art, which is why I had them quoted; but I think this is probably fine. I do think we should keep _discrimination threshold_ italicized though during the definition in the second sentence.